src/lib/fetch.ts

```diff
@@ -139,7 +139,7 @@ function toPermission(p: AuthRequired): Permission {
 type FetchedAccount = {
   publicKey: string;
   nonce: string;
-  tokenId: string;
+  token: string;
```

Comment:
this file has some unrelated fixes which turned up when debugging snarkyjs transactions against a local Mina node, while debugging timestamp preconditions

Reply:
Good catch on figuring out these fixes!